src/constructs/loadbalancing/elb.test.ts

```diff
@@ -331,24 +332,43 @@ describe("The GuHttpsApplicationListener class", () => {
 
     const json = SynthUtils.toCloudFormation(stack) as SynthedStack;
 
-    expect(json.Parameters.CertificateARN).toEqual({
+    expect(json.Parameters.TLSCertificate).toEqual({
       Type: "String",
-      AllowedPattern: "arn:aws:[a-z0-9]*:[a-z0-9\\-]*:[0-9]{12}:.*",
-      ConstraintDescription: "Must be a valid ARN, eg: arn:partition:service:region:account-id:resource-id",
+      AllowedPattern: RegexPattern.ACM_ARN,
+      ConstraintDescription: "Must be an ACM ARN resource",
       Description: "Certificate ARN for ApplicationListener",
     });
 
     expect(stack).toHaveResource("AWS::ElasticLoadBalancingV2::Listener", {
       Certificates: [
         {
           CertificateArn: {
-            Ref: "CertificateARN",
+            Ref: "TLSCertificate",
           },
         },
       ],
     });
   });
 
+  test("passing in an invalid ACM ARN", () => {
```

Comment:
Minor suggestion, it might be nice to describe the expected behaviour in the test name?

```suggestion
  test("throws an error if an invalid ACM ARN is passed in", () => {
```